src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const rightVersionOfRobot = {};
+
+  for (const key in robot) {
+    if (rightVersionOfRobot.hasOwnProperty(robot[key])) {
```

Comment:
robot[key] is used a couple of times, create a variable for it

Reply:
Thank you! 
Variable was added